wsagent/agent/src/main/resources/org.eclipse.che.ws-agent.script.sh

```diff
@@ -18,7 +18,11 @@ is_current_user_sudoer() {
 }
 
 set_sudo_command() {
-    if is_current_user_sudoer && ! is_current_user_root; then SUDO="sudo -E"; else unset SUDO; fi
+    if is_current_user_sudoer; then
```

Comment:
Changing this is an error. Function `is_current_user_sudoer` returns `true` if the user is `root`. As a consequence `SUDO` is set to `sudo -E"` when the script is run by `root` user. The correct behavior should be to `unset SUDO` if user is `root`. 

By the way the goal of your PR is to avoid executing `chown` and `chmod` if the user is arbitrary. For this you don't need to change this function (`set_sudo_command`).